src/libs/Clipboard/index.ts

```diff
@@ -59,23 +82,25 @@ function setHTMLSync(html, text) {
         // See https://dvcs.w3.org/hg/editing/raw-file/tip/editing.html#the-copy-command for more details.
     }
 
-    selection.removeAllRanges();
+    selection?.removeAllRanges();
 
     if (isComposer) {
-        firstAnchorChild.setSelectionRange(originalSelection.start, originalSelection.end, originalSelection.direction);
+        const composerSelection = originalSelection as ComposerSelection;
```

Comment:
instead of using `as ComposerSelection` you can do something like that 
```suggestion
        if ('start' in originalSelection && isComposer)
```
then typescript will know that originalSelection is type of ComposserSelection and else will be AnchorSelection

Reply:
done and Thanks